packages/synapse-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sage-bionetworks/synapse-client",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
release latest changes for @sage-bionetworks/synapse-client. One way to fix the bug in release 537 would be to update synapse-react-client to use this version once it's released to npm.